include/evmc/instructions.h

```diff
@@ -3,13 +3,19 @@
  * Licensed under the MIT License. See the LICENSE file.
  */
 
+/**
+ * EVM Instruction Tables
+ *
+ * A collection of metrics for EVM1 instruction set.
+ *
+ * @defgroup instructions EVM Instructions
+ * @{
+ */
 #pragma once
 
 #include <evmc/evmc.h>
 #include <evmc/utils.h>
 
-#include <stdint.h>
```

Comment:
Why remove this? `int16_t` still requires it.

Reply:
It's already included in `<evmc/evmc.h>`.